src/Badge.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+const type = props => {
+  switch(props.type) {
+    case 'success':
+      return {
+        backgroundColor: theme.colors.green,
+        color: theme.colors.white
+      }
+    case 'success-light':
+      return {
+        backgroundColor: theme.colors.green0,
+        color: theme.colors.black
+      }
+    case 'warning':
+      return {
+        backgroundColor: theme.colors.red,
+        color: theme.colors.white
+      }
+    case 'warning-light':
+      return {
+        backgroundColor: theme.colors.red0,
+        color: theme.colors.black
+      }
+    case 'attention':
+      return {
+        backgroundColor: theme.colors.orange,
+        color: theme.colors.white
+      }
+    case 'attention-light':
+      return {
+        backgroundColor: theme.colors.orange0,
+        color: theme.colors.black
+      }
+    case 'information':
+      return {
+        backgroundColor: theme.colors.blue,
+        color: theme.colors.white
+      }
+    case 'information-light':
+      return {
+        backgroundColor: theme.colors.blue0,
```

Comment:
Can `green0`, `red0`, `orange0` and `blue0 ` from legacy theme be replaced to use similar colors from new theme? If the namespaces don't exist yet in new theme, maybe we can refactor the same colors from legacy them to the new theme?